cucumber/authenticators_oidc/features/authn_oidc_performance.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Users can authneticate with OIDC authenticator
```

Comment:
if this is already new file , i would change the feature name to explain it for performance